telegraf/service/running.sls

```diff
@@ -15,3 +15,6 @@ telegraf/service/running:
     - enable: True
     - require:
       - sls: {{ sls_config_file }}
+    - reload: True
+    - watch:
+      - file: {{ sls_config_file }}
```

Comment:
```suggestion
      - sls: {{ sls_config_file }}
```

`sls_config_file` is the whole `sls`.

Reply:
I really don't now. I added since it worked for me.

Reply:
> When set, reload the service instead of restarting it.